modules/porous_flow/src/materials/PorousFlowConstantBiotModulus.C

```diff
@@ -0,0 +1,42 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "PorousFlowConstantBiotModulus.h"
+
+template<>
+InputParameters validParams<PorousFlowConstantBiotModulus>()
+{
+  InputParameters params = validParams<PorousFlowMaterialVectorBase>();
+  params.addRangeCheckedParam<Real>("biot_coefficient", 1.0, "biot_coefficient>=0 & biot_coefficient<=1", "Biot coefficient");
+  params.addRangeCheckedParam<Real>("fluid_bulk_modulus", 2.0E9, "fluid_bulk_modulus>=0", "Fluid bulk modulus");
```

Comment:
Probably should be strictly greater than zero since you divide by this quantity.